src/mame/drivers/r9751.cpp

```diff
@@ -136,13 +150,23 @@ class r9751_state : public driver_device
 	uint32_t debug_a6();
 	uint32_t debug_a5();
 	uint32_t debug_a5_20();
+	void UnifiedTrace(u32 address, u32 data, const char* operation="Read", const char* Device="SMIOC", const char* RegisterName=NULL, const char* extraText=NULL);
```

Comment:
Please use `nullptr` as the null pointer constant.

Reply:
Understood, will do.